### ISSUE TITLE
Use precompiled Erlang modules

### ISSUE THREAD

**Issue by tynanbe** [gleam-lang/mix_gleam#17]
`MixGleam` should use precompiled Erlang modules when Gleam projects provide them.

**Comment by lpil** [comment]
I don't think this is the job of MixGleam as it knows how to compile Gleam, there would be no advantage to using the precompiled Erlang. Instead we want this feature in Mix itself.

**Comment by tynanbe** [comment]
That makes sense, but MixGleam's detection of when to compile Gleam versus when Mix will use the precompiled Erlang might need updating once that happens.
